client/components/stats-interval-dropdown/index.jsx

```diff
@@ -23,7 +23,7 @@ const StatsIntervalDropdownListing = ( { selected, onSelection, intervals, onGat
 				},
 				{
 					name: 'jetpack_stats_upsell_clicked',
-					params: { statType: intervals[ interval ].statType, source: event_from },
+					params: { stat_type: intervals[ interval ].statType, source: event_from },
```

Comment:
Is this an intentional change? It might break consistency in stats data, wouldn't it?